doc/source/conf.py

```diff
@@ -67,29 +67,42 @@
 from IPython.core.inputtransformer2 import TransformerManager
 import re
 
+
 def ipython_cell_transform(source):
```

Comment:
We now have builtin [IPython support](https://sphinx-codeautolink.readthedocs.io/en/latest/examples.html#ipython-code-blocks) by the way!